src/utils/network/session-core.js

```diff
@@ -182,8 +170,9 @@ export const uploadDocument = async (
   const ttlFileExists = hasTTLFiles(datasetFromUrl);
 
   // Guard clause will throw function if container already exist with ttl file
+  // ttl file indicates update instead of upload document
   if (ttlFileExists) {
-    throw new Error('Container already exists');
+    throw new Error('TTL file already exists');
```

Comment:
It's fine if a container exist. However, a TTL file is the main indicator of whether or not a file has previously been uploaded.